bin/crew

```diff
@@ -1847,6 +1857,111 @@ def print_deps_tree(args)
   puts treeView
 end
 
+def upload(args)
+  abort "\nGITLAB_TOKEN environment variable not set.\n".lightred if ENV['GITLAB_TOKEN'].nil?
+
+  packages = ''
+  gitlab_token = ENV['GITLAB_TOKEN']
+
+  if ARGV.length > 0
+    ARGV.each do |package|
+      unless package == 'upload'
+        pkgfile = "#{CREW_LOCAL_REPO_ROOT}/packages/#{package}.rb"
+        if File.exist?(pkgfile)
+          packages += " #{package}"
+        else
+          puts "Package #{pkgfile} not found.".orange
+        end
+      end
+    end
+  else
+    %w[.tar.xz .tpxz .tar.zst].each do |ext|
+      packages += `find #{CREW_LOCAL_REPO_ROOT}/release/*/*#{ext} -exec basename -s #{ext} {} + | cut -d- -f1 | sort | uniq | xargs`
+      packages += ' '
+    end
+    abort 'No package binaries found.'.lightred if packages.empty?
+  end
+
+  base_url = 'https://gitlab.com/api/v4/projects/26210301/packages/generic'
+
+  packages.strip!
+
+  [packages].each do |package|
+    %w[x86_64 i686 armv7l].each do |arch|
+      puts "Package: #{package}, Arch: #{arch}".yellow
+      pkgfile = "#{CREW_LOCAL_REPO_ROOT}/packages/#{package}.rb"
+      new_tarfile = Dir["#{CREW_LOCAL_REPO_ROOT}/release/#{arch}/#{package}-*-chromeos-#{arch}.{tar.xz,tpxz,tar.zst}"].max_by { |f| File.mtime(f) }
```

Comment:
We aren't creating .tpxz files anymore.

Reply:
Doesn't hurt to include just in case.

Reply:
On the contrary-- if we are creating .tpxz files it's a bug, the installer will break (in its current state) and some of my scripts will break. 

The creation and uploading of new .tpxz files is a bug, and if we catch it at the upload stage it will save us hassle in the future.

Reply:
I would argue it's not a bug.  What is a bug (or lacking feature, rather) is not being flexible and allowing and coding for ANY compression format.  IOW, we shouldn't be setting limits.  In the future, there could be a better compression format that may be preferable.

Reply:
And in the future, we can switch to that. Flexibility may be easy to account for here, in ruby, but in the installer and in helper scripts it can make things much more complex.

essentially, imo we shouldn't be checking for binaries compressed in a format we deprecated